<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Генератор паролей</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_length">
      <property name="text">
       <string>Длина пароля:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBox_length">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>128</number>
      </property>
      <property name="value">
       <number>12</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_numbers">
      <property name="text">
       <string>Использовать цифры</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_uppercase">
      <property name="text">
       <string>Использовать заглавные буквы</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_symbols">
      <property name="text">
       <string>Использовать специальные символы</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_generate">
      <property name="text">
       <string>Сгенерировать пароль</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit_password">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>Здесь будет сгенерированный пароль</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_copy">
      <property name="text">
       <string>Копировать пароль</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_strength">
      <property name="text">
       <string>Введите пароль для оценки:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_evaluate">
      <property name="placeholderText">
       <string>Введите пароль</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_evaluate">
      <property name="text">
       <string>Оценить надёжность</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_result">
      <property name="text">
       <string>Надёжность: ---</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
